apple/RNGHStylusData.h

```diff
@@ -0,0 +1,82 @@
+//
+//  RNGHStylusData.h
+//  Pods
+//
+//  Created by Michał Bert on 18/09/2024.
+//
+
+#ifndef RNGHStylusData_h
+#define RNGHStylusData_h
+
+@interface RNGHStylusData : NSObject
+
+@property (atomic, assign) double tiltX;
+@property (atomic, assign) double tiltY;
+@property (atomic, assign) double altitudeAngle;
+@property (atomic, assign) double azimuthAngle;
+@property (atomic, assign) double pressure;
+
+- (NSDictionary *)toDictionary;
+
+@end
+
+typedef struct {
+  double tiltX;
+  double tiltY;
+} RNGHTilt;
```

Comment:
Wait, why not use `CGPoint`?

Reply:
True, we could use that!

Done in [b2a7437](https://github.com/software-mansion/react-native-gesture-handler/pull/3113/commits/b2a7437d18c51829dc29baf7cc62c65b3b419d9c)